src/app/design/styles/color/index.html

```diff
@@ -369,7 +369,7 @@
     <div class="definition-props">
       <div class="definition-usage">
         <div class="sky-field-label">Usage</div>
-        <div>Background for categorized containers or for containers that have no categories.</div>
+        <div>Background for categorized content.</div>
       </div>
       <div class="definition-code">
         <div class="sky-field-label">Code</div>
```

Comment:
Is it too late to rename the styles for the colors to use "categorization" or "category" instead of "container"?

Reply:
Nope. Done.